worlds/khrecom/Client.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import annotations
+import os
+import sys
+import asyncio
+import shutil
+import logging
+import time
+from calendar import timegm
+
+import ModuleUpdate
+ModuleUpdate.update()
+
+import Utils
+
+death_link = False
+
+item_num = 1
+
+logger = logging.getLogger("Client")
+
+if __name__ == "__main__":
+    Utils.init_logging("KHRECOMClient", exception_logger="Client")
+
+from NetUtils import NetworkItem, ClientStatus
+from CommonClient import gui_enabled, logger, get_base_parser, ClientCommandProcessor, \
+    CommonContext, server_loop
+
+
+def check_stdin() -> None:
+    if Utils.is_windows and sys.stdin:
+        print("WARNING: Console input is not routed reliably on Windows, use the GUI instead.")
+
+class KHRECOMClientCommandProcessor(ClientCommandProcessor):
+    def __init__(self, ctx):
+        super().__init__(ctx)
+
+    def _cmd_deathlink(self):
+        """Toggles Deathlink"""
+        global death_link
+        if death_link:
+            death_link = False
+            self.output(f"Death Link turned off")
+        else:
+            death_link = True
+            self.output(f"Death Link turned on")
+
+class KHRECOMContext(CommonContext):
+    command_processor: int = KHRECOMClientCommandProcessor
```

Comment:
Type hint here is an `int`?

Reply:
I've seen this incorrect hint in a bunch of worlds, not sure where people are copying it from